interfaces/console/Dockerfile

```diff
@@ -1,62 +1,27 @@
-# FROM nginx
-# COPY  build /usr/share/nginx/html
-# RUN rm /etc/nginx/conf.d/default.conf
-# COPY nginx/nginx.conf /etc/nginx/conf.d
-# EXPOSE 8080
-# CMD ["nginx", "-g", "daemon off;"]
+FROM node:18-alpine AS base
```

Comment:
Can you use node:alpine instead of node node:18 to reduce the sieze of the image?